JavaScript/JS-Week_1/Javascript_Homework/Goodboy-Oldboy/script.js

```diff
@@ -0,0 +1,15 @@
+
+const dogYearOfBirth = 2019;
+const dogYearFuture  = 2030;
+let dogYear = dogYearFuture - dogYearOfBirth ;
+let shouldShowResultInDogYears = true ;
+
+if (shouldShowResultInDogYears) {
+    dogYear = dogYear * 7;
+}
```

Comment:
You can remove this `if else` and move the `dogYear = dogYear * 7;` right before the `console.log` in the next `if else`. It will spare you a bit of repetition.